layouts/partials/notice.html

```diff
@@ -1,11 +1,22 @@
 {{ $page := .Page }}
 
-{{ range $i := $.Site.Data.notices }}
-{{ if eq $i.title $.Params.Title }}
-<div id="note" class="{{ $i.type | lower }}">
-  <h2>{{ $i.type }}</h2>
-  {{ $i.note | markdownify}}
-</div>
+{{ $notices := $.Site.Data.notices }}
+
+{{ $previewmessage := "This is a preview of an unreleased feature. This documentation is subject to change." }}
+
+
+{{if $page.Params.preview }}
+{{ $previewnotice := dict "note" $previewmessage "preview" "true" "type" "cta"}}
+{{ $notices = $notices | append $previewnotice }}
+{{ end }}
+
+{{ range $i := $notices }}
+{{ if or (eq $i.title $.Params.Title) (eq $i.preview "true") }}
+<section style="margin-bottom:-1em" class="section section-cta">
```

Comment:
This is a nit and out of scope of this PR, but I wonder if the CTAs can be a different color on dark mode. They don't stand out so much to me since it's the same color as the tags.

<img width="750" alt="dark mode cta" src="https://github.com/user-attachments/assets/2f5bb806-ee39-41d0-a34c-f900f6c4face">

Reply:
OK, we discussed this and sounds like you might pick out a color combo and do a quick PR on this :+1: